rest/utilities_testing.go

```diff
@@ -2802,3 +2802,26 @@ func AssertHTTPErrorReason(t testing.TB, response *TestResponse, expectedStatus
 
 	assert.Equal(t, expectedReason, httpError.Reason)
 }
+
+func AssertRevTreeAfterHLVConflictResolution(t *testing.T, doc *db.Document, expectedActiveRevTreeID, expectedTombstoneParentID string) {
+	activeLeafCount := 0
+	for _, revID := range doc.History.GetLeaves() {
+		revItem := doc.History[revID]
+		if revItem.Deleted {
+			assert.Equal(t, expectedTombstoneParentID, revItem.Parent)
+		} else {
+			activeLeafCount++
+			assert.Equal(t, expectedActiveRevTreeID, revID)
+		}
+	}
+	// should only have one active leaf
+	require.Equal(t, 1, activeLeafCount)
+}
+
+func StripInternalProperties(t *testing.T, body db.Body) db.Body {
```

Comment:
Do you want to make stripInternalProperties public from db or at least deduplicate the one in topologytests